src/api/app/views/webui2/webui/package/binaries.html.haml

```diff
@@ -0,0 +1,62 @@
+:ruby
+  @pagetitle = "State of Repository #{@repository} for #{@project} / #{@package_name}"
+  @metarobots = 'index,nofollow' # do not follow the single binary pages
+  package_bread_crumb 'Repository State'
+
+.card
+  = render partial: 'tabs', locals: { package: @package, project: @project }
+
+  .card-body
+
+    %h3= @pagetitle
+    %p
+      = render partial: 'shared/download_repository_link', locals: { project: @project, repository: @repository }
+      - unless @buildresults.empty?
+        = link_to sprited_text('drive_burn', 'Delete'), { controller: :package, action: :wipe_binaries,
+                                                          project: @project, package: @package_name, repository: @repository }, method: :delete
+        all built binaries
+    - @buildresults.each do |result|
+      %h3
+        = repo_status_icon(result[:repocode])
+        Architecture: #{result[:arch]}
+      - if result[:binaries].empty?
+        %p
+          %i No built binaries
+      - else
+        %ul
+          - result[:binaries].each do |binary|
+            %li.binaries_list_item
+              -# haml-lint:disable UnnecessaryStringOutput
```

Comment:
moving ```"#{binary[:filename]} (#{number_to_human_size(binary[:size])})"``` to ```#{binary[:filename]} (#{number_to_human_size(binary[:size])})``` doesn't output anything

Reply:
@vpereira, did you also remove the `=`?

Reply:
you could use `#{binary[:filename]} (#{number_to_human_size(binary[:size])})` instead of `= "#{binary[:filename]} (#{number_to_human_size(binary[:size])})"`  because HAML can make the insterpolation. [Here an example](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/views/webui2/webui/package/_buildstatus.html.haml#L7)